### ISSUE TITLE
O.NotNull and O.DBType not accessible in PostgresDriver tables

### ISSUE THREAD

**Issue by hvesalai** [slick/slick#1659]
The use of O.NotNull and O.DBType is documented in the documentation, but they cannot be used, for example, in PostgresDriver.

According to one stackoverflow post, this is inteded (http://stackoverflow.com/questions/39890665/cannot-resolve-symbol-notnull-with-slick-3-1-1) deprecation. If so, the documentation should be updated to reflect this.

```
value NotNull is not a member of driver.ColumnOptions
def total = column[Int]("total", O.NotNull)

value DBType is not a member of driver.ColumnOptions
def word = column[String]("word", O.DBType("TEXT"))
```

**Comment by hvesalai** [comment]
Or maybe my analysis was wrong and `O.DBType` is now `O.SqlType`

**Comment by cvogt** [comment]
NotNull should not be used manually. The correct way is

```
value NotNull is not a member of driver.ColumnOptions
def total = column[Option[Int]]("total")
```

**Comment by hvesalai** [comment]
So the documentation is out-dated?

http://slick.lightbend.com/doc/3.1.1/schemas.html#table-rows

**Comment by cvogt** [comment]
right, seems outdated. DBType => SqlType. Null/NotNull should probably be removed there. Mind giving it a shot :)?

**Comment by hvesalai** [comment]
This has been resolved by #1660
